### ISSUE TITLE
Terrible Loss when training on SSD MobileNet v2 320x320 

### ISSUE THREAD

**Issue by Rodeazzo** [tensorflow/models#9952]
Hi,

I've been struggling with finding a suitable setting to train on the aforementioned model. I tried modifying the learning_rate_base from its original .8 (which was giving me insane loss values), to 0.03, 0.2, 8e-2 and now 0.004. Pretty much the same occurrence happens as the image below:

![image](https://user-images.githubusercontent.com/33449183/116253837-1e835a80-a771-11eb-96f9-26c9721b85be.png)

Aside of learning rate, I changed warmup_learning_rate to 0.001, and use_dropout to true from convolutional_box_predictor.

Anyone has any idea why this is happening? Should I let it train for longer maybe it stabilises? 

Thanks

**Comment by Mrinal18** [comment]
Hey @Rodeazzo, could you share your code? its not possible to just judge it by looking at the learning rate or convolutional_box_predictor. Will take a look and let you know.

**Comment by Rodeazzo** [comment]
Here's a link to my Google Colab document:
https://colab.research.google.com/drive/19YKxOg2y4tEcIKeYrfl6TRkoPxmK_Aqw?usp=sharing
